docs/verifiable-payid.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: verifiable-payid
-title: Getting Started With Verifiable PayID
-sidebar_label: Verifiable PayID
+title: Enhance Security With Verifiable PayID
+sidebar_label: Enhance Security With Verifiable PayID
```

Comment:
Love this change. Great to have a tightly focused page just explaining the concept.